bin/republish_documents

```diff
@@ -2,8 +2,31 @@
 
 require File.expand_path("../../config/environment", __FILE__)
 require "specialist_publisher"
+require "logger"
 
-document_types = ARGV.any? ? ARGV : SpecialistPublisher.document_types
+logger = Logger.new(STDOUT)
```

Comment:
I think it's leaked from this PR (now merged): https://github.com/alphagov/specialist-publisher/pull/491